examples/images_contours_and_fields/colormap_normalizations_symlognorm.py

```diff
@@ -1,42 +1,85 @@
 """
 ==================================
-Colormap Normalizations Symlognorm
+Colormap Normalizations SymLogNorm
 ==================================
 
 Demonstration of using norm to map colormaps onto data in non-linear ways.
 
 .. redirect-from:: /gallery/userdemo/colormap_normalization_symlognorm
 """
 
+########################################
+# Synthetic dataset consisting of two humps, one negative and one positive,
+# the positive with 8-times the amplitude.
+# Linearly, the negative hump is almost invisible,
+# and it is very difficult to see any detail of its profile.
+# With the logarithmic scaling applied to both positive and negative values,
+# it is much easier to see the shape of each hump.
+#
+# See `~.colors.SymLogNorm`.
+
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.colors as colors
 
-"""
-SymLogNorm: two humps, one negative and one positive, The positive
-with 5-times the amplitude. Linearly, you cannot see detail in the
-negative hump.  Here we logarithmically scale the positive and
-negative data separately.
 
-Note that colorbar labels do not come out looking very good.
-"""
+def rbf(x, y):
+    return 1.0 / (1 + 5 * ((x ** 2) + (y ** 2)))
 
-N = 100
+N = 200
+gain = 8
 X, Y = np.mgrid[-3:3:complex(0, N), -2:2:complex(0, N)]
-Z1 = np.exp(-X**2 - Y**2)
-Z2 = np.exp(-(X - 1)**2 - (Y - 1)**2)
-Z = (Z1 - Z2) * 2
+Z1 = rbf(X + 0.5, Y + 0.5)
+Z2 = rbf(X - 0.5, Y - 0.5)
+Z = gain * Z1 - Z2
+
+shadeopts = {'cmap': 'PRGn', 'shading': 'gouraud'}
+colormap = 'PRGn'
+lnrwidth = 0.5
 
-fig, ax = plt.subplots(2, 1)
+fig, ax = plt.subplots(2, 1, sharex=True, sharey=True)
 
 pcm = ax[0].pcolormesh(X, Y, Z,
-                       norm=colors.SymLogNorm(linthresh=0.03, linscale=0.03,
-                                              vmin=-1.0, vmax=1.0, base=10),
-                       cmap='RdBu_r', shading='nearest')
+                       norm=colors.SymLogNorm(linthresh=lnrwidth, linscale=1,
+                                              vmin=-gain, vmax=gain, base=10),
+                       **shadeopts)
 fig.colorbar(pcm, ax=ax[0], extend='both')
+ax[0].text(-2.5, 1.5, 'symlog')
 
-pcm = ax[1].pcolormesh(X, Y, Z, cmap='RdBu_r', vmin=-np.max(Z),
-                       shading='nearest')
+pcm = ax[1].pcolormesh(X, Y, Z, vmin=-gain, vmax=gain,
+                       **shadeopts)
 fig.colorbar(pcm, ax=ax[1], extend='both')
+ax[1].text(-2.5, 1.5, 'linear')
+
+
+########################################
+# Clearly, tt may be necessary to experiment with multiple different
+# colorscales in order to find the best visualization for
+# any particular dataset.
```

Comment:
```suggestion
# In order to find the best visualization for any particular dataset,
# it may be necessary to experiment with multiple different color scales.
```
Swap the order to get rid of the "clearly"? and fix typo.